builder/builder.go

```diff
@@ -79,7 +79,7 @@ func GetBuilder(img imgutil.Image) (*Builder, error) {
 
 	label, err := img.Label(MetadataLabel)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to find run images for builder %s", style.Symbol(img.Name()))
+		return nil, errors.Wrapf(err, "failed to get metadata label from image %s", style.Symbol(img.Name()))
```

Comment:
We should probably give the label name in this error message so folks know what to look for.